docs/operations-guide/migrating-from-h2.md

```diff
@@ -15,15 +22,22 @@ export MB_DB_PORT=5432
 export MB_DB_USER=<username>
 export MB_DB_PASS=<password>
 export MB_DB_HOST=localhost
-java -jar metabase.jar load-from-h2 /path/to/metabase.db # do not include .mv.db or .h2.db suffix
+java -jar metabase.jar load-from-h2 /path/to/metabase.db # do not include .mv.db
 ```
 
 It is expected that you will run the command against a brand-new (empty!) database; Metabase will handle all of the work of creating the database schema and migrating the data for you.
 
-#### Notes
 
-- Avoid upgrading and migrating at the same time, since it can cause problems with one of database schemas not matching.
-- It is required that you can connect to the target MySQL or Postgres database in whatever environment you are running this migration command in. So, if you are attempting to move the data to a cloud database, make sure you take that into consideration.
-- For MySQL or MariaDB, the minimum recommended version is MySQL 5.7.7 and MariaDB 10.2.2, while the following is required: `utf8mb4_unicode_ci` collation, `utf8mb4` character set, and `innodb_large_prefix=ON`.
-- For PostgreSQL, the minimum version is PostgreSQL 9.4, since the code that handles these migrations uses a command that is only available in version 9.4 or newer.
-- H2 automatically adds a `.h2.db` or `.mv.db` extension to the database path you specify, so make sure the path to the DB file you pass to the command _does not_ include it. For example, if you have a file named `/path/to/metabase.db.h2.db`, call the command with `load-from-h2 /path/to/metabase.db`.
+### PostgreSQL notes
+
+-  Minimum version is PostgreSQL 9.4, since the code that handles these migrations uses a command that is only available in version 9.4 or newer.
+
+## MySQL/MariaDB notes
+
+- MySQL minimum recommended version is 5.7.7.
+- MariaDB minimum recommended version is 10.2.2.
+- And the following database settings are required, which is the default in the above recommended versions: `utf8mb4_unicode_ci` collation, `utf8mb4` character set, and `innodb_large_prefix=ON`.
+
+### Troubleshooting
+
+If you get stuck, check out [Error when loading application database from H2](../troubleshooting-guide/loading-from-h2.md).
```

Comment:
What does "get stuck" mean in this context? I.e, what are the symptoms someone should look for?